api/specs.proto

```diff
@@ -234,6 +246,14 @@ message NetworkSpec {
 	bool internal = 4;
 
 	IPAMOptions ipam = 5 [(gogoproto.customname) = "IPAM"];
+
+	// AllowContainers allows individual containers to be attached
+	// to this network. With this flag enabled individual
+	// containers running in an worker node in the cluster can
+	// attach to this network and access the services attached to
+	// this network. If this flag is not enabled(default case)
+	// independent containers cannot attach to this network.
+	bool allow_containers = 6;
```

Comment:
> global_network

The implications of this name are way too wide. Does that mean the network is globally visible?

> open_to_all_nodes

We may want to scope this in the future to a specific set of nodes.

> worker_visible

All networks are "visible" to the worker.

> allow_worker_attach

Will this feature always be limited to a worker? What if I want to have a host that can attach to these networks but not run workloads? That should be possible, without changes.

The term `Attachable` is strictly true. This is a network to which one can attach.